packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -412,11 +412,68 @@
     margin: 1.2rem 2.4rem;
     align-items: center;
     gap: 0.8rem;
+
+    &__balance {
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__list-of-accounts {
+        display: flex;
+        justify-content: space-between;
+        &:first-child {
+            border-top: 2px solid var(--general-section-1);
+            padding-top: 1rem;
+        }
+        margin: 0.8rem 0;
+    }
+
     &__acc_status {
         display: flex;
         flex-direction: column;
         align-items: flex-end;
     }
+
+    &__expansion-panel {
+        background-color: var(--general-section-5);
+        padding: 1.6rem 2.4rem;
+        border-radius: $BORDER_RADIUS;
+
+        &--content {
+            padding: 0.8rem 0;
+        }
+
+        &--divider {
+            width: 100%;
+            border-bottom: 2px solid var(--general-hover);
+        }
+    }
+
+    &__get-more-accounts {
+        cursor: pointer;
+        border-radius: 12px;
+        display: flex;
+        align-items: center;
+        border: 2px dashed var(--general-hover);
+        padding: 1rem 1.8rem;
+        margin: 1.6rem 0;
+
+        &--icon {
+            margin-right: 1.5rem;
+        }
+
+        &--details {
+            line-height: 0.9;
+            margin-top: 0.3rem;
+            &--title {
+                color: var(--text-general);
+                font-size: 1.4rem;
```

Comment:
@hirad-deriv can't we apply these styles to the Text component itself instead of here?